Sources/TuistDependencies/SwiftPackageManager/Utils/SwiftPackageManagerGraphGenerator.swift

```diff
@@ -231,3 +302,15 @@ extension PackageInfo.Target.Dependency.PackageConditionDescription {
         })
     }
 }
+
+extension PackageInfo.Target.TargetBuildSettingDescription.Setting {
+    public var extractDefine: (name: String, value: String) {
+        let define = self.value[0]
+        if define.contains("=") {
+            let split = define.split(separator: "=", maxSplits: 1)
+            return (name: String(split[0]), value: String(split[1]))
+        } else {
+            return (name: define, value: "1")
```

Comment:
It's the default value if you don't specify one, from [SPM doc](https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html#csetting):
```
/// Defines a value for a macro.
///
/// If you don't specify a value, the macro's default value is 1.
///
/// - Since: First available in PackageDescription 5.0
///
/// - Parameters:
///   - name: The name of the macro.
///   - value: The value of the macro.
///   - condition: A condition that restricts the application of the build setting.
static func define(_ name: String, to value: String? = nil, _ condition: BuildSettingCondition? = nil) -> CSetting
```